src/index.ts

```diff
@@ -110,16 +110,17 @@ export const htmlPlugin = (configuration: Configuration = { files: [], }): esbui
         return compiledTemplateFn(templateContext)
     }
 
-    function injectFiles(dom: JSDOM, assets: { path: string }[], outDir: string, htmlFileConfiguration: HtmlFileConfiguration) {
+    function injectFiles(dom: JSDOM, assets: { path: string }[], outDir: string, publicPath: string, htmlFileConfiguration: HtmlFileConfiguration) {
         const document = dom.window.document
         for (const outputFile of assets) {
             const filepath = outputFile.path
             const out = path.join(outDir, htmlFileConfiguration.filename)
             const relativePath = path.relative(path.dirname(out), filepath)
+            const absolutePath = publicPath + relativePath
```

Comment:
Couldn't publicPath also not end with a slash? This could cause some problems if just string concatenation is used.

One option would be to use a small library for this. I particularly like https://github.com/balazsbotond/urlcat.

Feel free to let me do this, if you don't want to do this by yourself. :)

Reply:
Thank you for commenting! I'm sorry for my carelessness.
I didn't know that library. I think this is great for our purpose! 
I'm sorry to trouble you, but could I ask you to fix this?

Reply:
No worries :) I will update the pull request.